rero_ils/modules/documents/views.py

```diff
@@ -637,3 +575,38 @@ def get_articles(record):
             'pid':hit.pid
         })
     return articles
+
+
+@blueprint.app_template_filter()
+def online_holdings(document_pid, viewcode='global'):
+    """Find holdings by document pid and viewcode.
+
+    : param document_pid: document pid
+    : param viewcode: symbol of organisation viewcode
+    : return: list of holdings
+    """
+    holdings = []
+    organisation = None
+    if viewcode != current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE'):
+        organisation = Organisation.get_record_by_viewcode(viewcode)
+    from ..holdings.api import HoldingsSearch
```

Comment:
move to first line of the function, just below the docstring

Reply:
done